libs/blocks/sticky-header/sticky-header.js

```diff
@@ -0,0 +1,24 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+
+export default function decorate($block) {
+  const $header = document.querySelector('header');
+  const $div = $block.firstElementChild;
+  const $a = $block.querySelector('a');
```

Comment:
what happens if there's a link in the text? You'd then be converting that link into a button.

Reply:
I have made changes to the selector. Please,  let me know if there is a better solution.